xbmc/pvr/timers/PVRTimers.cpp

```diff
@@ -469,9 +469,9 @@ bool CPVRTimers::InstantTimer(const CPVRChannel &channel)
   if (!g_PVRManager.CheckParentalLock(channel))
     return false;
 
-  CEpgInfoTag epgTag;
-  bool bHasEpgNow = channel.GetEPGNow(epgTag);
-  CPVRTimerInfoTag *newTimer = bHasEpgNow ? CPVRTimerInfoTag::CreateFromEpg(epgTag) : NULL;
+  CEpgInfoTagPtr epgTag(channel.GetEPGNow());
+  bool bHasEpgNow = (epgTag != NULL);
```

Comment:
Googled a little. There seems not to be 100% agreement on that stuff like <code>bool bHasEpgNow = (epgTag != NULL);</code> works, but agreement is that <code>bool bHasEpgNow = (epgTag.get != NULL);</code> or <code>bool bHasEpgNow = epgTag;</code> (although the latter looks strange, it works due to shared_ptr's operator bool) will work. Thus, I will rescan the PR and change all respective code places accordingly.